mlflow_project/local_environment/src/rf_test/train.py

```diff
@@ -76,47 +78,38 @@ def train(params, fpath, hyperopt=False):
         return _train(params, fpath, hyperopt)
 
 
-if (__name__ == "__main__"):
+if __name__ == "__main__":
```

Comment:
how do these train files relate to train and train_simple above in docker_environment?